packages/notifi-react-card/lib/components/subscription/SubscriptionCardV1.tsx

```diff
@@ -148,6 +148,45 @@ export const SubscriptionCardV1: React.FC<SubscriptionCardV1Props> = ({
     };
   }, [onClose]);
 
+  const useCustomTitles = data?.titles?.active === true;
+
+  const expiredHeader = () => {
+    return useCustomTitles && data?.titles.expiredView !== ''
+      ? data?.titles.expiredView
+      : copy?.expiredHeader ?? 'Welcome Back';
+  };
+
+  const previewHeader = () => {
+    return useCustomTitles && data?.titles.previewView !== ''
+      ? data?.titles.previewView
+      : copy?.manageAlertsHeader ?? 'Manage Alerts';
+  };
+
+  const signUpHeader = () => {
+    return useCustomTitles && data?.titles.signupView !== ''
+      ? data?.titles.signupView
+      : copy?.signUpHeader ?? 'Get Notified';
+  };
+  const editHeader = () => {
+    return useCustomTitles && data?.titles.editView !== ''
+      ? data?.titles.editView
+      : copy?.editHeader ?? 'Update Settings';
+  };
+  const verifyOnboardingHeader = () => {
+    console.log('data', data?.titles);
+    return useCustomTitles && data?.titles.verifyWalletsView !== ''
+      ? data?.titles.verifyWalletsView
+      : copy?.verifyWalletsHeader ?? 'Verify Wallets';
+  };
+
+  const historyView = () => {
+    return selectedAlertEntry === undefined
+      ? useCustomTitles && data?.titles.alertDetailsView !== ''
+        ? data?.titles.alertDetailsView
+        : copy?.historyHeader ?? 'Alert History'
+      : copy?.detailHeader ?? 'Alert Details';
+  };
+
```

Comment:
there's probably a clever Typescript-y way to DRY these up, but for now i think this is fine.

Reply:
there were some ways but i started getting errors where it was expecting true properties might not exist